rl_algorithms/common/networks/brain.py

```diff
@@ -206,3 +206,20 @@ def calculate_fc_input_size(self, state_dim: tuple):
         x = torch.zeros(state_dim).unsqueeze(0)
         output = self.backbone(x).detach().view(-1)
         return output.shape[0]
+
+
+class ACERBrain(Brain):
```

Comment:
Please use exsisting brain instead of making a new brain for acer. Divding actor and critic network might help.